examples/fatBeacon/simple.js

```diff
@@ -0,0 +1,20 @@
+// Simplest way to create a Eddystone-URL Beacon
+
+var eddystoneBeacon = require('./../../index');
+
+var html = "<html>" + 
+      "<head>" +
+        "<title>Fat Beacon Demo</title>" +
+        "<meta charset='UTF-8'>" + 
+        "<meta name='description' content='Fat Beacon Demo'/>" + 
+       "</head>" + 
+       "<body>" +
+        "<h1>HelloWorld</h1>" +
+        "This is a test" + 
+        "<div>Adding more content to see if I can it over the MTU value</div>" + 
+        "<div>Adding more content to see if I can it over the MTU value</div>" + 
+        "<div>Adding more content to see if I can it over the MTU value</div>" + 
+       "</body>" + 
+      "</html>";
+
```

Comment:
What do you think about adding this in a separate file and using the `fs` module to read the contents?